content/podcast/s02e08-season-finale/index.md

```diff
@@ -0,0 +1,98 @@
+- [integer 32 Rust consulting](https://www.integer32.com/)
+- [Clip 12: Arpad Borsos: Rust and Python integration with PyO3](https://corrode.dev/podcast/s01e06-sentry?t=57%3A16)
+- [Clip 13: Paul Dix: On hiring Rust engineers](https://corrode.dev/podcast/s01e01-influxdata/?t=46%3A49)
+- [Clip 14: Micah Wylde: Small companies can attract people who want to work in Rust](https://corrode.dev/podcast/s01e04-arroyo?t=29%3A42)
+- [Clip 15: Micah Wylde: Hiring Rust engineers in small vs big companies](https://corrode.dev/podcast/s01e04-arroyo?t=29%3A20)
+- [Clip 16: Folkert de Vries: Rust doesn't hide complexity and that's great](https://corrode.dev/podcast/s01e05-tweede-golf?t=13%3A18)
+- [Clip 17: Folkert de Vries: Understanding the Rust source code](https://corrode.dev/podcast/s01e05-tweede-golf?t=15%3A13)
+- [The Rust standard library source code](https://github.com/rust-lang/rust/tree/master/library)
+- [Crates.io all-time download charts](https://crates.io/crates?sort=downloads)
+- [Clip 18: Stephen Blum: The risk of overengineering solution in Rust is a barrier to entry](https://corrode.dev/podcast/s01e02-pubnub?t=37%3A24)
+- [Clip 19: Nicolas Moutschen: The three phases of Rust usage](https://corrode.dev/podcast/s01e03-apollo?t=24%3A09)
+- [`tokio` on crates.io](https://crates.io/crates/tokio)
+- [`axum` on crates.io](https://crates.io/crates/axum)
+- [`sqlx` on crates.io](https://crates.io/crates/sqlx)
+- [The 'New Rustacean' podcast &mdash; an inspiration for the show](https://newrustacean.com/)
+- ['On the Metal' podcast &mdash; another inspiration for the show](https://oxide.computer/podcasts/on-the-metal)
+- [Simon Brüggen (M3t0r) on GitHub](https://github.com/M3t0r)
+- [Simon Brüggen (M3t0r) on Mastodon](https://hachyderm.io/@m3t0r)
+- [Matthias Endler on Mastodon](https://mastodon.social/@mre)
+- [About corrode Rust Consulting](https://corrode.dev/about)
+- ['Rust in Production' Podcast on Apple](https://podcasts.apple.com/us/podcast/rust-in-production/id1720771330)
+- ['Rust in Production' Podcast on Spotify](https://open.spotify.com/show/0Hf6gWrzpSzXp1X0cebbsT)
+
+### Additional Links
+
+- [Rust 2023 Survey](https://blog.rust-lang.org/2023/12/20/Rust-2023-Survey-Results.html)
+- ['Rust in Production' title music (while you wait for season 2)](https://www.premiumbeat.com/royalty-free-tracks/cassiopeia)
+
+### All Episodes of Season 1
+
+If you missed any of the previous episodes, you can find them here:
+
+- [Episode 1: curl with Daniel Stenberg](/podcast/s02e01-curl)
+- [Episode 2: AMP with Carter Schultz](/podcast/s02e02-amp)
+- [Episode 3: Thunderbird with Brendan Abolivier](/podcast/s02e03-thunderbird)
+- [Episode 4: Matic with Eric Seppanen](/podcast/s02e04-matic)
+- [Episode 5: OxidOS with Alexandru Radovici](/podcast/s02e05-oxidos)
+- [Episode 6: Fusion Engineering with Jakub Valtar](/podcast/s02e06-fusion-engineering)
+- [Episode 7: System76 with Jeremy Soller](/podcast/s02e07-system76)
```

Comment:
We might have to change these :sweat_smile: 

There are some links at the end of the doc.

When these are updated, also update them in Let's Cast show notes!

Reply:
done